Gemfile

```diff
@@ -11,3 +11,4 @@ end
 gem 'jruby-openssl', :platform=> :jruby
 gem 'activemodel', ENV['RAILS_VERSION'] if ENV['RAILS_VERSION']
 gem 'linkeddata', '~> 1.99'
+gem 'rake', '< 12'
```

Comment:
Just pinning rake to < 12 seemed the easiest solution all around given that I just want to be able to use Rubydora 2.0.0.